adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -43,4 +43,6 @@
 
 @property (nonatomic, readonly, assign) BOOL muteIsSet;
 
+@property (nonatomic, assign) NSInteger nativeAdOptionPosition;
```

Comment:
Should there be a `NS_ENUM` for what the publisher can set?

See step 4 in https://support.vungle.com/hc/en-us/articles/4416477036955#configure-the-native-ad-for-display-using-storyboard-0-6.

And I would assume the publisher can set this via the code below (referring to [this dev doc](https://developers.google.com/admob/ios/mediation/vungle#network-specific_parameters)):
```
#import <VungleAdapter/VungleAdapter.h>
//...

GADRequest *request = [GADRequest request];
VungleAdNetworkExtras *extras = [[VungleAdNetworkExtras alloc] init];
extras.nativeAdOptionPosition = 1; // Specify the position of the privacy icon to be in the top left.
//...
[request registerAdNetworkExtras:extras];
```

Reply:
Yes, the example is correct. The publisher would set this through the network extras.
I didn't add an enum to the adapter code and followed the implementation of the `orientation` property, which uses the integer and translates to our enum value.

https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h#L40